### ISSUE TITLE
Scaling of glyphs to UPM?

### ISSUE THREAD

**Issue by davelab6** [Omnibus-Type/Jaldi#13]
Jaldi appears very small (left in the images below) compared to Ek Mukta (right) for Latin...:

![screen shot 2015-04-15 at 12 05 15 am](https://cloud.githubusercontent.com/assets/261579/7152111/2882b1cc-e303-11e4-83da-d1c2d3ac87ac.png)

![screen shot 2015-04-15 at 12 06 49 am](https://cloud.githubusercontent.com/assets/261579/7152121/571f66b0-e303-11e4-8a17-b1a84c483662.png)

**Comment by Omnibus-Type** [comment]
I will match the height the Devanagari glyphs of Jaldi with the height the Devanagari glyphs of EkMukta. As you can see in the attached image, EkMukta and Jaldi have different criteria to related the proportion of the Latin glyphs with the Devanagari glyphs. In conclusion, if I match the size of the Devanagari glyphs of Jaldi with EkMukta, the latin glyphs could seem smaller. Is that ok?

![jaldi-ekmuktametricscomparative](https://cloud.githubusercontent.com/assets/8041808/7162640/925407f8-e34a-11e4-94d5-cc0901b08684.png)

**Comment by davelab6** [comment]
I think the Latin could be scaled up so the cap heights of Ek Mukta and Jaldi are similar.

**Comment by davelab6** [comment]
Is that comparison using the current version of Jaldi?

**Comment by Omnibus-Type** [comment]
Yes, I am taking it from the cloned repository on my computer, that sample is the .otf

**Comment by davelab6** [comment]
This is weird... The deva subset looks the same size:

<img width="1090" alt="screen shot 2016-01-14 at 23 23 17" src="https://cloud.githubusercontent.com/assets/261579/12345366/dd6936b6-bb15-11e5-8717-f47a70e06128.png">

But the Latin does not:

<img width="1102" alt="screen shot 2016-01-14 at 23 23 08" src="https://cloud.githubusercontent.com/assets/261579/12345365/dd677a9c-bb15-11e5-89a6-57436851a868.png">

**Comment by Omnibus-Type** [comment]
It seems that since the latest message, Jaldi has not been scaled yet. I will talk with Pablo.
